tests/tests/cpld_restart_p7

```diff
@@ -51,34 +51,35 @@ export LIST_FILES="sfcf024.tile1.nc \
                    RESTART/sfc_data.tile5.nc \
                    RESTART/sfc_data.tile6.nc \
                    RESTART/MOM.res.nc \
-                   RESTART/iced.2016-10-04-00000.nc \
-                   RESTART/ufs.cpld.cpl.r.2016-10-04-00000.nc"
+                   RESTART/iced.2021-03-23-21600.nc \
+                   RESTART/ufs.cpld.cpl.r.2021-03-23-21600.nc"
 
 export_fv3
 export_cpl
 
-export FHROT='12'
+export FHROT=12
 export RESTART_N=$((FHMAX-$FHROT))
 
-export RESTART_FILE_PREFIX="${SYEAR}${SMONTH}${SDAY}.$(printf "%02d" $(( ${FHROT}  )))0000"
-export RESTART_FILE_SUFFIX_HRS="${SYEAR}-${SMONTH}-${SDAY}-$(printf "%02d" $(( ${FHROT} )))"
-export RESTART_FILE_SUFFIX_SECS="${SYEAR}-${SMONTH}-${SDAY}-$(printf "%02d" $(( ${FHROT}*3600 )))"
+export RESTART_FILE_PREFIX="${SYEAR}${SMONTH}${SDAY}.$(printf "%02d" $(( ${FHROT} + ${SHOUR} )))0000"
+export RESTART_FILE_SUFFIX_HRS="${SYEAR}-${SMONTH}-${SDAY}-$(printf "%02d" $(( ${FHROT} + ${SHOUR})))"
+export RESTART_FILE_SUFFIX_SECS="${SYEAR}-${SMONTH}-${SDAY}-$(printf "%02d" $(( (${FHROT}+ ${SHOUR})*3600 )))"
 
 # ATM warm start
-export WARM_START='.true.'
-export MAKE_NH='.false.'
-export NA_INIT='0'
-export EXTERNAL_IC='.false.'
-export NGGPS_IC='.false.'
-export MOUNTAIN='.true.'
+export WARM_START=.true.
+export MAKE_NH=.false.
+export NA_INIT=0
+export EXTERNAL_IC=.false.
+export NGGPS_IC=.false.
+export MOUNTAIN=.true.
+export NSTF_NAME=2,0,0,0,0
```

Comment:
Is there a reason why NSTF_NAME in cpld_restart_p7 is different from that defined in default_vars.sh (2,1,0,0,0) which is used by cpld_control_p7?

Reply:
In restart, the setting is 2,0,0,0 because there should be no spin-up on a restart run.  See issue [695](https://github.com/ufs-community/ufs-weather-model/issues/695https://github.com/ufs-community/ufs-weather-model/issues/695).

Reply:
I think since the new ICs from GFSv16 (2021032206) do have the nst fields, we don't need to use (2,1,0,0,0) to spin up nsst model for coldstart either, we can directly use (2,0,0,0,0) so this NSTF_NAME does not need to change for restart. But ICs for benchmark runs does not have the nst  fields, we do need the spin-up (2,1,0,0,0).